README.md

```diff
@@ -72,31 +72,52 @@ const vc = require('vc-js');
 const suite = new Ed25519Signature2018({
   verificationMethod: publicKey.id,
   key: new Ed25519KeyPair({privateKeyBase58})
-})
+});
 
 vc.issue({ credential, suite })
     .then(issuedVc => console.log)
-    .catch(console.error)
+    .catch(console.error);
 ```
 
-### `verify()`
+### Creating a Verifiable Presentation - `createPresentation()`
+
+To create a Verifiable Presentation out of one or more Verifiable Credentials:
+
+```js
+const vc = require('vc-js');
+
+const verifiableCredential = [vc1, vc2];
+
+const suite = new Ed25519Signature2018({
+  verificationMethod: publicKey.id,
+  key: new Ed25519KeyPair({privateKeyBase58})
+})
+
+vc.createPresentation({ verifiableCredential, suite })
```

Comment:
Should all these calls be consistent with our normal style rules?  (ie, remove the space padding).

Reply:
Fixed.